### ISSUE TITLE
Feedback: Sind alle relevanten persönlichen Information vorhanden?

### ISSUE THREAD

**Issue by sirblackheart** [sirblackheart/elektrasurvey#53]
>  Wenig 'persoenliche' angaben noetig -> alle die relevant waeren vorhanden?

**Comment by markus2330** [comment]
evt. gender noch..

**Comment by sirblackheart** [comment]
Land könnte interessant

**Comment by markus2330** [comment]
Ja, stimmt. Gibts da vielleicht irgendein geoIP feature von limesurvey so dass Sie nur anhacken wollen ob Sie es mitteilen wollen?

**Comment by Namoshek** [comment]
Wir können IPs loggen, von geoIP hab ich noch nichts gehört. Glaube auch nicht, dass LimeSurvey da eine Auswertung zur Verfügung stellt (wären ziemlich viele API calls bei einer Auswertung...). Denke, dass das IP-Logging zur Verhinderung von Survey-Manipulation dient.

Was versprecht ihr euch von der Information des Landes?
Gender finde ich nicht gut, hatte ja schon in unsrem Meeting gesagt, dass ich das als (eh schon in der Unterzahl befindende) Frau eher abschreckend empfinden würde (und bei einer so speziellen Umfrage kann das auch schnell ein negatives Gefühl vermitteln, wenn man nicht viel ausfüllen kann und sich das auf die Statistik auswirkt).

**Comment by sirblackheart** [comment]
Land könnte von dem her interessant sein ob bestimmte Vorlieben geografisch bedingt sind. z.B. durch die ansässigen FLOSS Communites. Ich erinnere mich an etwas ältere Statistiken wo es sehr starke Unterschiede zwischen Deutschland und der USA bei der Verbreitung von KDE und GNOME gab.

> Gender finde ich nicht gut, hatte ja schon in unsrem Meeting gesagt

Bin auch nicht begeistert von der Gender Frage. Auch wenn gerne in der Statistiken Mann/Frau vergleiche hergenommen werden um Unterschiede aufzuzeigen finde ich es eher dem Zeitgeist entsprechend dem Entgegenzuwirken.

**Comment by markus2330** [comment]
Ok, dann lassen wir die Gender Frage. Ich habe es nur nochmals erwähnt weil eine Expertin extra darauf hingewiesen hat.

Wenn man die IP Adresse hat, können wir auch im Postprocessing von IP auf Länder auflösen. Sollen wir das einfach am Beginn erwähnen dass wir es machen, machen wir es von der E-Mail abhängig, oder machen wir eine eigene Frage ob wir die IP für die Länderzuweisung verwenden dürfen. (Da könnte man dann auch ein Other: Feld anbieten um das Land zu korrigieren falls jemand z.B. TOR verwendet).

**Comment by Namoshek** [comment]
Wenn dann Letzteres. Über die Email ist zu ungenau und zu mühsam; Ersteres könnte dazu führen, dass uns Teilnehmer abspringen, wenn sie es wissen.

**Comment by markus2330** [comment]
Wir müssten also noch eine Single Choice Frage im privaten Teil:

Do you want to share location-information with us?

- Yes, please use my IP Adress
- No, please do not use my location-information
- Yes, I am from <textfield>

Ideal wäre eine Weltkarte zum klicken. Alternativ könnte man versuchen die Adresse im Textfeld über APIs wie https://wiki.openstreetmap.org/wiki/Nominatim auflösen.

Was meint Ihr?

**Comment by markus2330** [comment]
Eine andere übliche Frage ist noch "Höchste abgeschlossene Schulbildung".

**Comment by Namoshek** [comment]
I think that choosing a location is not possible, that would require a completely new question type. We can offer a textbox for the country (or be crazy enough to create a drop-down with all countries). We could also use a textfield and ask for something else, like the country code or phone country code (it's less likely that they provide different versions of the same number than it would probably be the case with answers like "United States" vs. "United states of america" vs. "USA" for example, which makes post-processing more difficult).

About the IP-Question: We cannot enable IP-logging separately for each question. It's either logging all or nothing. But of course we could use the question to determine if we may use the IP in our post-processing (which still will be quite some work, especially because I've no idea how to do that efficiently). Therefore I think asking directly for their country code or something else is a better solution.

Last question: Is ok for me. Any ideas what good (international) answers could be? I always have a hard time figuring out which ones I have to select when coming across international ones...

PS: Sorry for writing in english, I just realized this issue was in german after writing the whole post.

**Comment by markus2330** [comment]
Dont worry about writing in english. "Höchste abgeschlossene Schulbildung" see #94

GeoIP is as simple as it could be: You use `geoiplookup <ip-addr>` and get `AT, Austria` (e.g. for my IP Address). I think there will be hundrets of problems if we let people enter this by hand. But a hybrid approach might be best: if we have the geoip and what they wrote we can easily fix problems and still have their consent. (for the people that filled out the textfield). In conflicting cases the manual would win, and if its unsure we would make NA or ask them by e-mail.
